runtime/runtime/src/state_viewer/errors.rs

```diff
@@ -0,0 +1,81 @@
+#[derive(thiserror::Error, Debug)]
+pub enum ViewAccountError {
+    #[error("Account ID #{requested_account_id} is invalid")]
+    InvalidAccountId { requested_account_id: near_primitives::types::AccountId },
+    #[error("Account ID #{requested_account_id} does not exist")]
+    AccountDoesNotExist { requested_account_id: near_primitives::types::AccountId },
+    #[error("Storage error: #{storage_error}")]
+    StorageError {
+        #[from]
+        storage_error: near_primitives::errors::StorageError,
+    },
+}
+
+#[derive(thiserror::Error, Debug)]
+pub enum ViewContractCodeError {
+    #[error("Account ID #{requested_account_id} is invalid")]
+    InvalidAccountId { requested_account_id: near_primitives::types::AccountId },
+    #[error("Account ID #{requested_account_id} does not exist")]
+    AccountDoesNotExist { requested_account_id: near_primitives::types::AccountId },
+    #[error("Storage error: #{storage_error}")]
+    StorageError {
+        #[from]
+        storage_error: near_primitives::errors::StorageError,
+    },
```

Comment:
I would move `StorageError` to be the last or even consider immediately convert it to `InternalError { error_message: String }` (or `InternalError { #[from] storage_error: StorageError }` wherever possible, and only `ViewAccessKeyError` will have it as a `String`). I don't have a strong preference here. Either way, I would put it the last variant on the list, and definitely name it as an `InternalError`, as from the external user point of view, that is the only thing they care about.